DS and Algorithm/Selenium/WebDriverSetup.java

```diff
@@ -1,13 +1,10 @@
-// WebDriverSetup.java
+// src/main/java/WebDriverSetup.java
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
 public class WebDriverSetup {
     public static WebDriver initializeChromeDriver() {
-        // Set the path to the ChromeDriver executable
-        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");
-        
-        // Create a new instance of the ChromeDriver
+        System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider making ChromeDriver path configurable</b></div>
<div id="fix">

Consider making the ChromeDriver path configurable through environment variables or a configuration file instead of hardcoding `lib/chromedriver.exe`. This would improve flexibility across different environments. A similar issue was also found in DS and Algorithm/Selenium/test.java (line 11).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```
 @@ -6,7 +6,8 @@
      public static WebDriver initializeChromeDriver() {
 -        System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
 +        String driverPath = System.getProperty("webdriver.chrome.path", "lib/chromedriver.exe");
 +        System.setProperty("webdriver.chrome.driver", driverPath);
          return new ChromeDriver();

```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/30#issuecomment-2535798608>#650ddd</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged